include/aspect/postprocess/gravity_point_values.h

```diff
@@ -232,13 +237,12 @@ namespace aspect
 
         /**
          * Specify the sampling scheme determining if gravity calculation is performed
-         * for a uniform distribution of points or a list of points.
+         * for a equianfgled map, uniformly distributed map or a list of points..
```

Comment:
I would simply remove the options (they are listed in the possible values). That way you do not need to change the description every time a new option is added.

`Specify the sampling scheme determining where gravity calculation is performed.` should be sufficient.